Sources/SourceKitLSP/Clang/ClangLanguageServer.swift

```diff
@@ -203,6 +224,17 @@ actor ClangLanguageServerShim: ToolchainLanguageServer, MessageHandler {
 
     process.standardOutput = clangdToUs
     process.standardInput = usToClangd
+    let logForwarder = ClangdStderrLogForwarder()
+    let stderrHandler = Pipe()
+    stderrHandler.fileHandleForReading.readabilityHandler = { fileHandle in
+      let newData = fileHandle.readData(ofLength: 1)
```

Comment:
Can't we just use `availableData` here?

Reply:
According to you on https://github.com/apple/swift-stress-tester/commit/4250ceb3d67d80d0f57943aa10122e026935c383

> Also using `readData` rather than `availableData` as various online comments have mentioned `availableData` causing various issues.

Do you remember what these *various issues* are? Or should we give `availableData` a try again?

And I can’t use `fileHandle.readData(ofLength: Int.max)` because that will block until it reaches `Int.max` bytes or the maximum size of `NSData`, which is usually more than a single line. Thus, `readData(ofLength: 1)` is the best I came up with.

Reply:
> Do you remember what these various issues are?

Hah, I knew I remembered something about this when I posted this comment. I believe it was something to do with `availableData` being empty even though there was more to read, so the `readabilityHandler` would be discarded before it actually should be. I really should have just added the various URLs there 😅 

Anyway, I vote we try `availableData` and come back to it if we see any problems.

Reply:
Changed it to `availableData`